### PR TITLE
[entropy_src/dv] UNR updates and toggle coverage closure

### PR DESCRIPTION
**Pull request by martin-lueker**
- New vRefine capturing results of a fresh UNR run with `entropy_src` at V2
   - Does not include most `prim_count` err_o outputs, which are indeed unreachable by allowed transisions, but should be tested by forcing tests.  (Exception: `prim_count`s within the SHA block are assumed already tested)
        - For now the `err_o` toggle exclusions are left in as a separate file to demonstrate 100% toggle coverage
- Separate exclusion file for `prim_count` outputs which will take forever to cover in simulation.
- Add `prim_count` related exclusions for ExtHT related counters

Since this PR starts with a fresh `--cov-unr` run, the diff may be hard to decipher, so here are some (hopefully helpful) analysis results:
- Almost of the rules remain the same with changes to the xml fields: `im-checksum`, `file` (numeric value), and `line`.   Masking these lines and comparing sorted text allows for comparison between this new UNR refine and the previous one.
- Almost all of the changes are related to toggle exclusions on one of the `err_o` outputs on one of the `prim_count`s.  All of these are technically unreachable, but forcing transitions in the regression in the `entropy_src_functional_errors` test run before the UNR analysis may cause some to be listed as `reachable` in the UNR analysis.   Differences in coverage for some runs may cause differences between two UNR runs for these constraints.
    - Those `prim_counts` that are merely used for reporting statistics are not (currently) forced in the `entropy_src_functional_errors` test and so will always be "unreachable".
- There is _one_ new UNR exclusion that has no correspondence in the previous file:
```
 <rule ccType="inst" ccfFlagsMask="2199136515010-42" domain="icc" entityName="entropy_src/u_reg/u_regwen/5" entityType="block" excTime="1668527928" file="15" im-checksum="1389479417" line="56" name="exclude" reviewer="unknown" text="end else if (wr_en) begin" user="1" vscope="default"></rule>
```

Please note that though some exclusions have been _removed_ from the UNR refine file.  All _additions_ have been saved to separate files. 

Signed-off-by: Martin Lueker-Boden <martin.lueker-boden@wdc.com>

### CODE REVIEW

**Comment by martin-lueker** [comment]
Regression results after adding this commit to master (d6ce715c6)

|  Score  |  Block  |  Branch  |  Statement  |  Expression  |  Toggle  |   Fsm   |  Assertion  |  CoverGroup  |
|:-------:|:-------:|:--------:|:-----------:|:------------:|:--------:|:-------:|:-----------:|:------------:|
| 97.94 % | 99.09 % | 97.48 %  |   99.67 %   |   95.14 %    | 100.00 % | 97.87 % |   91.29 %   |   93.37 %    |

**Review by weicaiyang — APPROVED** [review]
LGTM, thanks Martin

**Review by andreaskurth — APPROVED** [review]
LGTM, thanks @martin-lueker!

(I haven't reviewed the `entropy_src_UNR.vRefine` diff but trust your description of the changes. :slightly_smiling_face:)

**Review by mwbranstad — APPROVED** [review]
Coverage numbers look great, LGTM

File: hw/ip/entropy_src/dv/entropy_src_sim_cfg.hjson [hunk 1]
```diff
@@ -42,10 +42,16 @@
...
   reseed: 50
 
-  xcelium_cov_refine_files: ["{proj_root}/hw/ip/entropy_src/dv/cov/entropy_src_UNR.vRefine",
-    // TODO(V3): Finalize coverage on ExtHT ports,
-    //           & Remove the following exclusion file
-    "{proj_root}/hw/ip/entropy_src/dv/cov/entropy_src_extht_exclusions.vRefine"]
+  xcelium_cov_refine_files: [
+  // TODO(V3): Finalize coverage on ExtHT ports & remove the following exclusion file
+  "{proj_root}/hw/ip/entropy_src/dv/cov/entropy_src_extht_exclusions.vRefine"
+  // TODO: Expand functional_errors test to force errors in the following counters
```
[the remark this answers is not in the document]
Comment (martin-lueker) [reply hw/ip/entropy_src/dv/entropy_src_sim_cfg.hjson]: Good point. I've changed this line.  I'll merge it after the CI is done again.

**Pull request merged**